README.md

```diff
@@ -1,2 +1,82 @@
-# BenchView
-A solution to view and keep track of benchmarking results in C#
+# GlassView
+
+A solution to view and keep track of benchmarking results of dotnet projects.
+
+This is still a project very much in it's infancy. But when it matures, it is supposed to export [BenchmarkDotNet](https://github.com/dotnet/BenchmarkDotNet) results such that they can be tracked over time. This should - if all goes according to plans -- allow teams to keep track of the performance characteristics of a project. Just as many teams already do with correctness and quality aspects by keeping trach of test results (coverage) and code quality metrics.
```

Comment:
```suggestion
This is still a project very much in it's infancy. But when it matures, it is supposed to export [BenchmarkDotNet](https://github.com/dotnet/BenchmarkDotNet) results such that they can be tracked over time. This should allow teams to keep track of the performance characteristics of a project, similarly like keeping track of the test results (coverage) and code quality metrics.
```

Reply:
Thanks for the simplification and clearer wording. I adopted it with subtle adjustments.